src/register/data/tests/reducers.test.js

```diff
@@ -92,6 +185,19 @@ describe('Registration Reducer Tests', () => {
     expect(reducer(state, action)).toEqual({ ...defaultState });
   });
 
+  it('should take back data during form reset', () => {
```

Comment:
do we need this test? The state is already with default values before this action.

Reply:
we have written because line was uncovered in reducer that is why we had to write it.

Reply:
If the purpose is to increase the coverage of the action, we need to set some default value in the state then dispatch this action and verify the state should go on empty state.

Reply:
Addressed